### DISCUSSION TITLE
Cava shader that expands bars at higher frequencies

### DISCUSSION THREAD

**Discussion by Xanazf** [Alexays/Waybar#3384] [Q&A]
hey

i noticed in the Cava section there's a showcase of Waybar with the Cava widget, and Cava expanded the bars as the frequency increased. Does anyone know how to do that?

https://github.com/Alexays/Waybar/wiki/Module:-Cava#example

**Comment by LukashonakV** [comment]
Hi @Xanazf , need to build waybar with the cava support or contact with your OS community to check whether exists waybar package for your distributive with cava module enabled. Then configure module cava as all others
